portfolio/src/main/webapp/script.js

```diff
@@ -60,7 +109,8 @@ function createCommentElement(comment) {
   commentElement.className = 'comment';
 
   const commentBody = document.createElement('span');
-  commentBody.innerText = comment.body;
+  const commentToDisplay = comment.email + ': ' + comment.body;
```

Comment:
nit - you don't need this one variable and can use it inline, e.g.
`commentBody.innerText = comment.email + ': ' + comment.body;`